### ISSUE TITLE
Collection "angular-cli-ghpages" cannot be resolved

### ISSUE THREAD

**Issue by IamKayrox** [angular-schule/angular-cli-ghpages#121]
Tried the command  `ng add angular-cli-ghpages`
I got this error:
```
[error] Error: Collection "angular-cli-ghpages" cannot be resolved.
at NodeModulesEngineHost.resolve (C:\Users\Kayrox\Documents\Projects\GitHub Page\node_modules\@angular-devkit\schematics\tools\node-module-engine-host.js:75:19)
at NodeModulesEngineHost._resolveCollectionPath (C:\Users\Kayrox\Documents\Projects\GitHub Page\node_modules\@angular-devkit\schematics\tools\node-module-engine-host.js:80:37)
at NodeModulesEngineHost.createCollectionDescription (C:\Users\Kayrox\Documents\Projects\GitHub Page\node_modules\@angular-devkit\schematics\tools\file-system-engine-host-base.js:110:27)
at SchematicEngine._createCollectionDescription (C:\Users\Kayrox\Documents\Projects\GitHub Page\node_modules\@angular-devkit\schematics\src\engine\engine.js:148:40)
at SchematicEngine.createCollection (C:\Users\Kayrox\Documents\Projects\GitHub Page\node_modules\@angular-devkit\schematics\src\engine\engine.js:141:43)
at AddCommand.getCollection (C:\Users\Kayrox\Documents\Projects\GitHub Page\node_modules\@angular\cli\models\schematic-command.js:128:35)
at AddCommand.runSchematic (C:\Users\Kayrox\Documents\Projects\GitHub Page\node_modules\@angular\cli\models\schematic-command.js:298:50)
at AddCommand.executeSchematic (C:\Users\Kayrox\Documents\Projects\GitHub Page\node_modules\@angular\cli\commands\add-impl.js:185:31)
at AddCommand.run (C:\Users\Kayrox\Documents\Projects\GitHub Page\node_modules\@angular\cli\commands\add-impl.js:151:21)
at async AddCommand.validateAndRun (C:\Users\Kayrox\Documents\Projects\GitHub Page\node_modules\@angular\cli\models\command.js:135:28)
at async Object.runCommand (C:\Users\Kayrox\Documents\Projects\GitHub Page\node_modules\@angular\cli\models\command-runner.js:198:24)
at async default_1 (C:\Users\Kayrox\Documents\Projects\GitHub Page\node_modules\@angular\cli\lib\cli\index.js:62:31)
```
Running on:
```
Angular CLI: 10.0.8
Node: 14.7.0
OS: Windows 10 x64

Angular: 10.0.14
... animations, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router
Ivy Workspace: Yes

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.1000.8
@angular-devkit/build-angular     0.1000.8
@angular-devkit/build-optimizer   0.1000.8
@angular-devkit/build-webpack     0.1000.8
@angular-devkit/core              10.0.8
@angular-devkit/schematics        10.0.8
@angular/cli                      10.0.8
@ngtools/webpack                  10.0.8
@schematics/angular               10.0.8
@schematics/update                0.1000.8
rxjs                              6.5.5
typescript                        3.9.7
webpack                           4.43.0
```

**Comment by JohannesHoppe** [comment]
Please try to delete your `package-lock.json` and your `node_modules` folder. Then do an `npm install` and try again to install the builder via `ng add angular-cli-ghpages`. Please report if this works for you.

**Comment by IamKayrox** [comment]
Did that and now it works. I checked my ´package.json´ and ´angular-cli-ghpages´ was installed already. I used the ´ngh´ command following guide before landing here, because of that, installed the package before trying the instructions here. I don't know if this was the cause. Another possible issue I spotted was that for some reason my ´npm´ installation defaulted to production mode, because of this, ´npm´ wouldn't install development packages.
